EventsExpress/Validation/UserMoreInfoCreateViewModelValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EventsExpress.Validation
+{
+    using EventsExpress.ViewModels;
+    using FluentValidation;
+
+    public class UserMoreInfoCreateViewModelValidator : AbstractValidator<UserMoreInfoCreateViewModel>
+    {
+        public UserMoreInfoCreateViewModelValidator()
+        {
+            RuleFor(x => x.UserId).NotEmpty().WithMessage("Name is required");
```

Comment:
We usually do not validate for UserId, but  if you want to check if it exists - use appropriate message.